src/redux/players.js

```diff
@@ -0,0 +1,76 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const perPage = 100; // Set the number of results per page
+const maxRetries = 5; // Increase the maximum number of retries
+
+const initialState = {
+  data: [],
+  status: 'idle',
+  error: null,
+};
+
+export const fetchPlayers = createAsyncThunk('players/fetchPlayers', async () => {
+  let allPlayers = [];
+  let page = 1;
+  let retries = 0;
+
+  while (retries < maxRetries) {
+    try {
+      // eslint-disable-next-line no-await-in-loop
```

Comment:
- [ ] As mentioned in the previous review, we are asked to avoid disabling linters and that linters helps us write clean and readable code. Let us kindly address this